src/context.ts

```diff
@@ -47,8 +47,9 @@ export interface CacheConfiguration {
 }
 
 export interface SubgraphConfiguration {
```

Comment:
what about making this configuration actually be a map of chainIds to strings, so it is already extensible whenever we add a new chain?

Reply:
Think it's a bit easier for consumers to just pass in values rather than a map, but yeah it could be more extendable like that, although not as consistent with the other Context values.

Reply:
@FiboApe gonna resolve this to merge it, but happy to discuss more, or see an example implementation if we want to change it